### ISSUE TITLE
Sickle Cell App

### ISSUE THREAD

**Issue by tjchmielewski** [ConsenSys/web3studio-projects#51]
Mobile dapp that provides features for Sick Cell care that can connect patients to specialists and help them with their day-to-day as they age 

**Belief statement:**
I believe sickle cell patients will be healthier and prone to less crisis because the app will be set up to check up that they are up to date on medication, appointments, nutrition, and hydration as well as connected to professionals remotely and near their area. 

## The Problem
* People with the Sickle cell trait are often not  
*The disease is relatively rare, so most hospitals and medical providers do not know how to manage the disease
* When a SCD patient goes to the emergency room, they are often stereotyped as junkies looking for drugs
* When SCD patient goes to the hospital, the longer they wait in the ER to get admitted, the more of a chance they will need a longer hospital stay
* SCD patients struggle to hold jobs because if their illness is not managed, they can end up in the hospital for 2 weeks at a clip and sporadically throughout the year.
* SCD patients often withdraw from society because they are dealing with their pain

Who is the target audience (daily routine, where are they, what are their goals)
*People with sickle cell (roughly 80,000 in US)
* Family members of sickle cell patients
* Specialists of SCD
* Testing Labs
* Hospital Registries
* Bone Marrow donors

**Comment by brian-lc** [comment]
I'm wondering if this should be an app at all, at least not one for someone with SCD. Would the belief statement still be true if it was a person who called/txt'd someone with SCD? 
>>>  "I believe sickle cell patients will be healthier and prone to less crisis because a [person will call/text] to check up that they are up to date on medication, appointments, nutrition, and hydration as well as connected to professionals remotely and near their area."

I don't know much about the community of SCD but from my experience at GrandRounds having a person to talk to about your chronic disease, an advocate, really can make an impact towards positive health outcomes. 

So perhaps the 'app' is for the support person to use to reach out to SCD sufferers in a secure way (provided the patient has opt'd in and is sharing their HIPAA protected info).

**Comment by tjchmielewski** [comment]
I agree. Saying "app" is a bit presumptuous. In my head, there's a mechanism connecting a professional or non-profit caretaker to a patient. There are triggers for both endpoints to check-up on a patient or for the patient to quickly say they are ok or not ok as well and have someone be able to keep track of their wellbeing.
